pkg/controllers/dynakube/activegate/internal/statefulset/builder/modifiers/eec.go

```diff
@@ -0,0 +1,81 @@
+package modifiers
+
+import (
+	dynatracev1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/activegate/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/activegate/internal/statefulset/builder"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/container"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ volumeModifier = EecModifier{}
+var _ volumeMountModifier = EecModifier{}
+var _ builder.Modifier = EecModifier{}
+
+const (
+	eecVolumeName = "eec-token"
+	eecDir        = "/var/lib/dynatrace/secrets/eec"
```

Comment:
please set it to "/var/lib/dynatrace/secrets/eec/token"

Reply:
@aorcholski If you already changed this, please close the discussions to make the PR more readable